Grow-673 borrower profile frame

BoulderBrains opened the pull request:
[Grow-673](https://kiva.atlassian.net/browse/GROW-673)

Borrower profile page frame with tailwinds grid, margin & max-width across screen sizes. 

Storybook story included as well.

mcstover commented on src/pages/BorrowerProfile/BorrowerProfile.vue:
```diff
@@ -0,0 +1,38 @@
+<template>
+	<!-- Header goes here -->
```
We just need to add the www-page component added to get the header and footer and all other global elements of the Ui app.

BoulderBrains replied:
I removed  header and footer comments

mcstover commented:
@emuvente @BoulderBrains I keep getting the same updates for package-lock.json so it may be fine to let them through here.

BoulderBrains commented:
@mcstover I reached out to @emuvente and cleared the package-lock.json changes earlier. They were a result of running 'npm ci', which I had to run in order to get storybook running again after merging the recent changes.

emuvente commented on src/pages/BorrowerProfile/BorrowerProfile.vue:
```diff
@@ -0,0 +1,35 @@
+<template>
```
Please add `www-page` as the root element (e.g. https://github.com/kiva/ui/blob/master/src/pages/BorrowerProfile/fundedBorrowerProfile.vue#L2). That will add the header and footer to the page.

BoulderBrains replied:
Just added.

eddieferrer commented:
> @emuvente @BoulderBrains I keep getting the same updates for package-lock.json so it may be fine to let them through here.

Same for me

BoulderBrains commented:
@mcstover, @emuvente & @eddieferrer, the package-lock.json file changes have been reverted from this PR.

emuvente commented on src/pages/BorrowerProfile/BorrowerProfile.vue:
```diff
@@ -0,0 +1,47 @@
+	<www-page
+		id="borrower-profile"
+		:header-theme="headerTheme"
+		:footer-theme="footerTheme"
```
Since we are using the default themes for this page, you don't need to include the header-theme and footer-theme props. Removing them will also take care of these warnings in the console:
```
[Vue warn]: Invalid prop: type check failed for prop "headerTheme". Expected Object, got String with value "".
[Vue warn]: Invalid prop: type check failed for prop "footerTheme". Expected Object, got String with value "".
```

emuvente reviewed (approved):
👍 looks great!

BoulderBrains commented:
One odd thing I've noticed in storybook for this new page frame is that there's an additional padding added to the 'sb-show-main sb-main-padded' which results in padding going around the header and footer. This same class is on other page files, but not respected the same way. 

<img width="1316" alt="Screen Shot 2021-06-01 at 10 21 33 AM" src="https://user-images.githubusercontent.com/1521381/120366591-b15d6a80-c2c4-11eb-8c87-2f93627ccee0.png">

ryan-ludwig commented on src/pages/BorrowerProfile/BorrowerProfile.vue:
```diff
@@ -0,0 +1,42 @@
+		max-w-6xl = max-width: 72rem;
+
+		-->
+		<div class="grid-cols-12 mx-5 md:mx-8 lg:mx-16 md:max-w-4xl lg:max-w-6xl flex-auto">
```
I believe a wrapper `<div class="container">` will remove the need for these classes.

BoulderBrains replied:
@ryan-ludwig Does the container class have the tailwinds grid, max-widths and margins set on it? Trying to better understand how this is all coming together.

ryan-ludwig reviewed (approved):
👍

mcstover commented on src/pages/BorrowerProfile/BorrowerProfile.vue:
```diff
@@ -0,0 +1,33 @@
+		<div class="container">
+			<div class="grid grid-cols-12 gap-x-2">
+				<div class="col-span-12 md:col-start-2 md:col-span-10 lg:col-span-7"
+					style="background-color:lightgreen;"
```
are these just for debugging + temporary grid visibility?

BoulderBrains replied:
I put them there, so you can see the grid breaking across screen sizes if you pulled it up in storybook. I'll remove the inline style before merging.

BoulderBrains commented:
I've reworked how the grid is applied. Adding the .container div gave me max-width and margin definitions. Within that container i've added three divs with tailwinds grids defined. To the two inner divs, there's an inline style so you can see the grid breaking across screen sizes, will remove before merging.

BoulderBrains commented:
> One odd thing I've noticed in storybook for this new page frame is that there's an additional padding added to the 'sb-show-main sb-main-padded' which results in padding going around the header and footer. This same class is on other page files, but not respected the same way.
> 
> **New page frame**
> <img alt="Screen Shot 2021-06-01 at 10 21 33 AM" width="1316" src="https://user-images.githubusercontent.com/1521381/120366591-b15d6a80-c2c4-11eb-8c87-2f93627ccee0.png">
> 
> **Another page with same elements, css is there but not respected the same way**
> <img alt="Screen Shot 2021-06-01 at 10 37 35 AM" width="1259" src="https://user-images.githubusercontent.com/1521381/120367303-8d4e5900-c2c5-11eb-8662-a2d68c7b5862.png">
> 
> * There is a margin: -1rem; on the < div > directly inside the < div id="root" > which corrects this padding for this page.

@ryan-ludwig what do you think about this situation I'm still seeing in storybook?

ryan-ludwig commented:
@BoulderBrains For the margin/fullscreen issue, you can add a layout parameter of 'fullscreen' to the story to get it to go to the edges. https://storybook.js.org/docs/react/configure/story-layout

Something like
```
// BorrowerProfile.stories.js

export default {
	title: '/Pages/Borrower Profile',
	component: BorrowerProfile,
	parameters: {
		layout: 'fullscreen',
	},
...
```

It wouldn't be a good idea to do that in all of the "Pages" stories, but we can come back around to it after this PR.
